nntrainer/src/databuffer_file.cpp

```diff
@@ -250,6 +249,7 @@ int DataBufferFromDataFile::setDataFile(std::string path, DataType type) {
     train_name = path;
   } break;
```

Comment:
IMO, there is a bug. Need to add `validation[type] = false;` and remove `train_name = path;` code from `if` block

Reply:
My mistake. LGTM!